suite-native/device/src/middlewares/deviceMiddleware.ts

```diff
@@ -43,7 +44,10 @@ export const prepareDeviceMiddleware = createMiddlewareWithExtraDeps(
          expect that the state was already changed by the action stored in the `action` variable. */
         next(action);
 
-        if (deviceActions.createDeviceInstance.match(action)) {
+        if (
+            deviceActions.createDeviceInstance.match(action) &&
+            action.payload.id !== PORTFOLIO_TRACKER_DEVICE_ID
+        ) {
```

Comment:
This is not correct IMO. We want portfolio tracker to be allowed to be selected. We just need to enable actual device to be selected if there already is a selected device which is a flow that's not yet been explored.

Reply:
we want to be able to select fake device for sure. But IMO it should be selected by some user Action (e.g. selecting it from the screen header or navigating from crossroads) . It should not be selected automatically from the `deviceReducer` after its initialization every time.